lib/escape_utils/html/cgi.rb

```diff
@@ -1,11 +1 @@
-class CGI
-  extend ::EscapeUtils::HtmlSafety
```

Comment:
So trying this branch in our app, quite a few things are broken with double escaping. 

So we should keep the part of the patch that adds support for `String#html_safe?`.

Reply:
Ok, so I restored the monkey patches, but solely for their `ActiveSupport::SafeBuffer` integration.